lib/widgets/drive.coffee

```diff
@@ -51,18 +68,45 @@ getDrives = ->
 # visuals.drive('Please select a drive').then (drive) ->
 # 	console.log(drive)
 ###
+
 module.exports = (message = 'Select a drive') ->
 	getDrives().then (drives) ->
-
-		if _.isEmpty(drives)
-			throw new Error('No available drives')
-
-		form.ask
-			type: 'list'
-			name: 'drive'
+		options =
 			message: message
+			name: 'drive'
 			choices: _.map drives, (drive) ->
 				return {
 					name: "#{drive.device} (#{drive.size}) - #{drive.description}"
 					value: drive.device
+					disabled: undefined
 				}
+
+		ui = new UI
+			input: process.stdin
+			output: process.stdout
+		list = new InquirerList(options, ui.rl)
+		scanner = new DriveScanner getDrives,
+			interval: 4000
+			drives: drives
+
+		scanner.on 'add', (drive) ->
+		###
```

Comment:
Can you use a single `#` in front of every line instead of `###` for consistency?